streaming/src/main/scala/org/apache/spark/streaming/api/python/PythonDStream.scala

```diff
@@ -48,6 +59,16 @@ private[python] trait PythonTransformFunction {
 private[python] trait PythonTransformFunctionSerializer {
   def dumps(id: String): Array[Byte]
   def loads(bytes: Array[Byte]): PythonTransformFunction
+
+  /**
+   * After invoking `dumps` or `loads`, the user should use `getLastFailure` to check if there is
+   * any failure in Python. If so, the user should handle properly, e.g., throw an exception with
```

Comment:
nit: Very verbose. Just simply say "Get the failure, if any, in the last call to `dumps` or `loads`. Returns the failure message if there was a failure, or `null` if there was not failure."